build-support/bin/generate_docs.py

```diff
@@ -493,27 +451,30 @@ def sync(self) -> None:
 
         All pages live under the "reference" category.
 
-        There are three top-level pages under that category:
+        There are four top-level pages under that category:
         - Global options
         - The Goals parent page
         - The Subsystems parent page
+        - The Targets parent page
 
-        The individual pages for each goal/subsystem are nested under the two parent pages.
+        The individual reference pages are nested under these parent pages.
         """
         # Docs appear on the site in creation order.  If we only create new docs
         # that don't already exist then they will appear at the end, instead of in
         # alphabetical order. So we first delete all previous docs, then recreate them.
         #
+        # TODO: Instead of deleting and recreating, we can set the order explicitly.
```

Comment:
What do you mean?

Reply:
There is an `order` field, so I think we can just update the doc with a new value for that field. But need to experiment with that.